docs/DEPLOYMENT.md

```diff
@@ -67,11 +109,13 @@ The command above will create a container image and push it to the registry poin
 Apply the release yaml
 
 ```bash
-ko apply -f api.yaml
+kubectl apply -f api.yaml
 ```
 
 The command above will create the deployment for Database and API and a job for DB-Migration.
 
+The db-migration job will create all tables in the db.
+
 ```bash
 $ kubectl get pods
```

Comment:
Since we are creating this in a namespace, so can we add `-n <namespace>` in this command?